passport-reshuffle/src/strategy.ts

```diff
@@ -0,0 +1,76 @@
+import { Profile, Strategy as StrategyType } from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import { Strategy as Auth0Strategy } from 'passport-auth0';
+import { titlecase } from 'stringcase';
+import { stickFigure } from './stick-figure';
+
+// Currently the *base* Passport Profile.  But could be anything
+// supported by the underlying Passport strategy.
+export { Profile };
+
+function getMissingOauthEnvariables() {
+  return ['OAUTH_DOMAIN', 'OAUTH_CLIENT_ID', 'OAUTH_CLIENT_SECRET']
```

Comment:
My preference while we have control over those variables is to prefix them with `RESHUFFLE_`, since they are very close to what a user might use for their other, unrelated oauth https://github.com/reshufflehq/dev-docs/blob/master/backend/authBackend.js#L17.

Reply:
Talk tomorrow.  I'm not sure which is better TBH, I'd like to poll some more people.